applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -117,6 +127,156 @@ static void vcs_flags_cb_handler(struct bt_vcs *vcs, int err, uint8_t flags)
 	}
 }
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+#if (CONFIG_BT_VCS_CLIENT)
+/**
+ * @brief  Callback handler for VCS discover finished
+ *
+ *         This callback handler will be triggered when VCS
+ *         discovery is finished.
+ */
+static void vcs_discover_cb_handler(struct bt_vcs *vcs, int err, uint8_t vocs_count,
+				    uint8_t aics_count)
+{
+	if (err) {
+		LOG_ERR("VCS discover finished callback error: %d", err);
+	} else {
+		LOG_DBG("VCS discover finished");
+	}
+}
+#endif /* (CONFIG_AUDIO_DEV == GATEWAY) */
```

Comment:
Nitpick: The order of these #endif comments should be reversed

Reply:
Will rather remove CONFIG_AUDIO_DEV == GATEWAY, no need to check both